SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -23,18 +23,16 @@ inline AActor* GetTopmostOwner(const AActor* Actor)
 {
 	check(Actor != nullptr);
 
-	AActor* Owner = Actor->GetOwner();
-	if (Owner == nullptr || Owner->IsPendingKillPending())
-	{
-		return nullptr;
-	}
+	AActor* Itr = const_cast<AActor*>(Actor);
 
-	while (Owner->GetOwner() != nullptr && !Owner->GetOwner()->IsPendingKillPending())
+	AActor* Owner = Actor->GetOwner();
+	while (Owner != nullptr && !Owner->IsPendingKillPending() && Owner->GetIsReplicated())
```

Comment:
Can we rename the function? I suggest this since now we're only iterating to the outermost owner along a chain of replicated owners.

Reply:
GetOutermostReplicatedOwner?

Reply:
Outer has a specific meaning in Unreal (meaning the package the actor is loaded in). I don't think we should be using it in this function name.

Reply:
GetTopmostReplicatedOwner
Is that what it was changed from?